src/activity-logger/background/index.js

```diff
@@ -9,3 +9,68 @@ browser.webNavigation.onCommitted.addListener(({url, tabId, frameId}) => {
     maybeLogPageVisit({url, tabId})
 
 })
+
+
+//********************************************************************************/
+var timestamp = new Date().getTime;
+
+var number_of_opened_tabs = 0;
+
+var tabList = [];
+
+
+/*chrome.runtime.onMessage.addListener(function(response,sender,sendResponse){
+    alert(response);
+});*/
+
+
+browser.tabs.onCreated.addListener(function (tab){
+    number_of_opened_tabs = number_of_opened_tabs + 1;
+    //console.log('total opened tabs: ' + number_of_opened_tabs);
+    var t = {
```

Comment:
this would be a const, but t isn't a great name. You can find a good name or push directly the object.